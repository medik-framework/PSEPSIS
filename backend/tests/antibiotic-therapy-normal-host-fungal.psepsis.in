[
  {
    "id": "tabletApp",
    "action": "broadcast",
    "eventName": "AgeEntered",
    "eventArgs": [500]
  },
  {
    "id": "tabletApp",
    "action": "broadcast",
    "eventName": "WeightEntered",
    "eventArgs": [14],
    "waitFor": {
      "name": "ConfirmAgeEntered",
      "args": []
    }
  },
  {
    "id": "tabletApp",
    "action": "broadcast",
    "eventName": "StartAntibioticTherapy",
    "eventArgs": [],
    "waitFor": {
      "name": "ConfirmWeightEntered",
      "args": []
    }
  },
  {
    "id": "tabletApp",
    "action": "broadcast",
    "eventName": "AntibioticsPrimaryConditionsEntered",
    "eventArgs": [false, false, false, false, false, false],
    "waitFor": {
      "name": "Instruct",
      "args": ["get antibiotics primary conditions"]
    }
  },
  {
    "id": "tabletApp",
    "action": "broadcast",
    "eventName": "FungalConcernEntered",
    "eventArgs": [],
    "waitFor": {
      "name": "RecommendDrug"
    }
  },
  {
    "action":"exit",
    "waitFor": {
      "name": "RecommendDrug"
    }
  }
]
