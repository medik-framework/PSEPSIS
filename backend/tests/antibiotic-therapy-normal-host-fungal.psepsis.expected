[
  {
    "id": "tabletApp",
    "interface": "TabletApp",
    "name": "ConfirmAgeEntered",
    "args": []
  },
  {
    "id": "tabletApp",
    "interface": "TabletApp",
    "name": "ConfirmWeightEntered",
    "args": []
  },
  {
    "id": "tabletApp",
    "interface": "TabletApp",
    "name": "Instruct",
    "args": ["get antibiotics primary conditions"]
  },
  {
    "id": "tabletApp",
    "interface": "TabletApp",
    "name": "RecommendDrug",
    "args": [
        "Children > 28 days who are normal hosts",
        "OSF Antibiotics Set",
        "drug combination",
        "vancomycin (15 mg/kg, maximum 1 to 2 g, for the initial dose)",
        "PLUS cefotaxime (100 mg/kg, maximum 2 g, for the inital dose) OR ceftriaxone (75 mg/kg, maximum 2 g, for the initial dose)"
    ]
  },
  {
    "id": "tabletApp",
    "interface": "TabletApp",
    "name": "RecommendDrug",
    "args": [
      "Patients at increased risk of fungal infection",
      "OSF Antibiotics Set",
      "drug combination",
      "Add liposomal amphotericin B or an echinocandin (e.g., caspofungin, micafungin) to the antimicrobial regimen"
    ]
  }
]

